Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ruby:2.2.1
+
+ENV APP_HOME /thredded
+RUN mkdir $APP_HOME
+WORKDIR $APP_HOME
+ADD . $APP_HOME
+
+RUN apt-get update -qq && \
```

Comment:
Typically you'll want to do steps like this one, that change very infrequently and also take a lot of time and bandwidth, earlier in the Dockerfile. In this ordering, any time you change any file in the repo, this `apt-get install` will have to be re-run. If you place it before the `ADD` line instead, the results of the `apt-get` command will be cached and re-used, until you change a preceding line or add to the list of required APT packages.